src/ssl.c

```diff
@@ -51519,24 +51942,30 @@ void wolfSSL_RSA_free(WOLFSSL_RSA* rsa)
 #ifdef HAVE_EX_DATA_CLEANUP_HOOKS
         wolfSSL_CRYPTO_cleanup_ex_data(&rsa->ex_data);
 #endif
-#if defined(OPENSSL_EXTRA_X509_SMALL) || defined(OPENSSL_EXTRA)
+#if defined(OPENSSL_EXTRA) || defined(OPENSSL_ALL)
```

Comment:
We loose `OPENSSL_EXTRA_X509_SMALL` here. Is that changed on purpose?

Reply:
Reverting back.